app/assets/javascripts/utilities/buildArticleHTML.js.erb

```diff
@@ -97,39 +118,41 @@ function buildArticleHTML(article) {
       }
     }
 
-    var readingTimeHTML = '';
-    if (article.class_name === "Article") {
-      // we have ` ... || null` for the case article.reading_time is undefined
-      readingTimeHTML = '<a href="'+article.path+'" class="article-reading-time">'+ ((article.reading_time || null) < 1 ? '1 min' : article.reading_time + ' min') +' read</a>'
-    }
-
     var videoHTML = '';
     if (article.cloudinary_video_url) {
-      videoHTML = '<a href="'+article.path+'" class="single-article-video-preview" style="background-image:url('+article.cloudinary_video_url+')"><div class="single-article-video-duration"><img src="<%= asset_path("video-camera.svg") %>" alt="video camera">'+article.video_duration_in_minutes+'</div></a>'
+      videoHTML = '<a href="'+article.path+'" class="crayons-story__video" style="background-image:url('+article.cloudinary_video_url+')"}><span class="crayons-story__video__time">'+article.video_duration_in_minutes+'</span></a>'
     }
 
     var timeAgoInWords = '';
     if (article.published_at_int) {
       timeAgoInWords = timeAgo({ oldTimeInSeconds: article.published_at_int });
     }
 
-    return '<div class="single-article single-article-small-pic" data-content-user-id="'+article.user_id+'">\
+    return '<div class="crayons-story" data-content-user-id="'+article.user_id+'">\
       '+videoHTML+'\
-      '+orgHeadline+'\
-      <div class="small-pic">\
-       <a href="/'+profileUsername+'" class="small-pic-link-wrapper">\
-         <img src="'+picUrl+'" alt="'+profileUsername+' profile" loading="lazy">\
-       </a>\
-       </div>\
-       <a href="'+article.path+'" class="small-pic-link-wrapper index-article-link" id="article-link-'+article.id+'">\
-        <div class="content">\
-         <h3>'+flareTag+filterXSS(article.title)+'</h3>\
-         '+searchSnippetHTML+'\
+      <div class="crayons-story__body">\
+        <div class="crayons-story__top">\
+          '+meta+'\
+          <button class="crayons-btn crayons-btn--ghost crayons-story__overflow" type="button">\
+            <svg class="crayons-icon" width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path fillRule="evenodd" clipRule="evenodd" d="M7 12a2 2 0 11-4 0 2 2 0 014 0zm7 0a2 2 0 11-4 0 2 2 0 014 0zm5 2a2 2 0 100-4 2 2 0 000 4z"/></svg>\
+          </button>\
+          <div class="crayons-dropdown">\
+            <a href="/" class="crayons-link crayons-link--block">Subscribe</a>\
```

Comment:
This will require a bit more that being a link. I imagine this is an AJAX call that would need to have this change to `Unsubscribe` if successful and probably an error message potentially.

Reply:
Definitely! It’s more a placeholder right now:)